mlflow/types/llm.py

```diff
@@ -425,6 +441,10 @@ def __setattr__(self, name, value):
     "stop": ["\n"],
     "n": 1,
     "stream": False,
+    "metadata": {
```

Comment:
do we need to put metadata in input example ?

Reply:
we don't need to, i was thinking that it might be an easy way to let people know it's supported, but yeah it could be confusing